bundles/org.openhab.binding.guntamatic/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,596 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="guntamatic"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="biostar">
+
+		<label>Guntamatic Biostar</label>
+		<description>Guntamatic Biostar Heating System</description>
+
+		<channels>
+			<channel id="controlBoilerApproval" typeId="controlBoilerApproval">
+				<label>Set Boiler Approval</label>
+			</channel>
+			<channel id="controlProgram" typeId="controlProgram">
+				<label>Set Program</label>
+			</channel>
+			<channel id="controlHeatCircProgram0" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 0</label>
```

Comment:
A channel label must be very short, ideally 2 or 3 words with each word started with an uppercase letter.
As an example for this channel, the label could rather be "Heat Circle 0 Program".
My comment is applicable for all your channels and all your thing types.

Reply:
There is one channel in order to set the program of the Heat Circle as well as one channel to read it. Therefore I would like to keep the "Set" in the label.
Would `Set Heat Circle 0 Program` still be acceptable?

Reply:
Yes if you cannot find something better.

Reply:
👍 updated, incl. readme